release/setup_test_branches.js

```diff
@@ -48,17 +48,18 @@ const testVersionDefault = VERSION
 
 const templatesPackageJsons = [
     './AndroidIDPTemplate/package.json',
-    './MobileSyncExplorerReactNative/package.json',
     './AndroidNativeKotlinTemplate/package.json',
-    './MobileSyncExplorerSwift/package.json',
     './AndroidNativeTemplate/package.json',
-    './iOSIDPTemplate/package.json',
     './HybridLocalTemplate/package.json',
-    './iOSNativeSwiftTemplate/package.json',
-    './iOS13NativeSwiftTemplate/package.json',
     './HybridRemoteTemplate/package.json',
-    './iOSNativeTemplate/package.json',
-    './ReactNativeTemplate/package.json'
+    './MobileSyncExplorerReactNative/package.json',
+    './MobileSyncExplorerSwift/package.json',
+    './ReactNativeTemplate/package.json',
+    './iOS13NativeSwiftTemplate/package.json',
+    './iOSIDPTemplate/package.json',
+    './iOSNativeSwiftEncryptedNotificationTemplate/package.json',
```

Comment:
This one was missing.
This script setup_test_branches.js, creates branches (dev2 and master2) for dry run of a release.
For things to work correctly, all "repo pointers" (in submodules and package.json) are changed from forcedotcom/some-repo to some-user/some-repo.